commands/sync_import_users.go

```diff
@@ -0,0 +1,94 @@
+package commands
+
+import (
+	"errors"
+	"os"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mmetl/services/data_integrity"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var SyncImportUsersCmd = &cobra.Command{
+	Use:   "sync-import-users",
+	Short: "Checks if any users in the export file already exist in the Mattermost instance, and makes sure both username and email are the same in both cases.",
+	Long:  "Checks if any users in the export file already exist in the Mattermost instance, and makes sure both username and email are the same in both cases. This requires credentials or to have local mode enabled/available.",
+	RunE:  syncImportUsersCmdF,
+}
+
+func init() {
+	SyncImportUsersCmd.Flags().StringP("file", "f", "", "the mmetl file to check")
+	SyncImportUsersCmd.Flags().StringP("output", "o", "", "the output file name")
+	SyncImportUsersCmd.Flags().Bool("update-users", false, "Whether to update user records in the import file. When this flag is not provided, the tool runs a dry run to note which users would be updated.")
```

Comment:
Taking into account that we're asking for an `output` file and not overwriting the contents of the original `file`, I'd rather make the default behavior update the users and provide with a more standard `--dry-run` flag